tests/fixtures.ts

```diff
@@ -73,7 +73,7 @@ export const test = baseTest.extend<TestFixtures, WorkerFixtures>({
         command: 'node',
         args: [path.join(__dirname, '../cli.js'), ...args],
       });
-      const client = new Client({ name: 'test', version: '1.0.0' });
```

Comment:
Check if it doesn't exist yet

Reply:
Or make it an array?

Reply:
We don't support multiclient mode and this PR is not trying to add one.